apps/labs/posts/release-spyder-4beta2.md

```diff
@@ -0,0 +1,157 @@
+---
+
+It has been almost two months since I joined Quansight in April, to start 
+working on Spyder maintenance and 
+development. So far, it has been a very exciting and rewarding journey under 
+the guidance of long time Spyder maintainer
+[Carlos Córdoba](https://github.com/ccordoba12).
+This is the first of a series of blog posts we will be writing to showcase 
+updates on the development of Spyder, new planned features and news on the 
+road to Spyder 4.0 and beyond.
+
+First off, I would like to give a warm welcome to
+[Edgar Margffoy](https://github.com/andfoy),
+who recently joined Quansight and will be working with the Spyder team to
+take its development even further. Edgar has been a core Spyder developer 
+for more than two years now, and we are very excited to have his (almost)
+full-time commitment to the project.
+
+# Spyder 4.0 Beta 2 released!
+
+Since August 2018, when the first beta of the 4.x series was released, the
+Spyder development team has been working hard on our next release.
+Over the past year, we've implemented the long awaited full-interface dark theme;
+overhauled our entire code completion and linting architecture to use the
+[Language Server Protocol](https://microsoft.github.io/language-server-protocol/),
+opening the door to supporting many other languages in the future;
+added a new Plots pane to view and manage the figures generated by your code;
+and numerous other feature enhancements, bug fixes and internal improvements.
+
+## Dark theme
+
+A full-interface dark theme has been a
+[long awaited feature](https://github.com/spyder-ide/spyder/issues/2350),
+and is enabled by default in Spyder 4. You can still select the
+light theme under  ``Preferences > Appearance`` by either choosing a light-background
+syntax-highlighting scheme, or changing ``Interface theme`` to ``Light``.
+
+![Screenshot of the Spyder main window with default panes, with the dark theme applied across the entire interface.](/posts/release-spyder-4beta2/spyder-qdarkstyle.png)
+
+Pretty, right :-) ?
+
+This enhancement was made possible by the work of
+[Colin Duquesnoy](https://github.com/ColinDuquesnoy),
+[Daniel Pizzeta](https://github.com/DPizzeta)
+and their [QDarkStyle](https://github.com/ColinDuquesnoy/QDarkStyleSheet/)
+package. The Spyder team is now actively collaborating with Colin and Daniel
+to pursue the release of QDarkStyle 3.x, which will be using Spyder's
+[QtSASS](https://github.com/spyder-ide/qtsass/)
+package to harness the power of SASS/SCSS and allow users to fully customize
+the theme dynamically.
+
+
+## Language Server Protocol architecture
+
+The Language Server Protocol (LSP) was created by Microsoft for Visual Studio Code to standardize how
+development tools (e.g. editors and IDEs) communicate with
+servers that provide code completion, linting and related facilities for
+different programming languages. With LSP, as
+[they describe it](https://microsoft.github.io/language-server-protocol/):
```

Comment:
The linked website doesn't have the original quote anymore. @goanpeca would you like to replace it with [an internet archive link](https://web.archive.org/web/20190528210156/https://microsoft.github.io/language-server-protocol/) that has the original quote, or keep it as is?
CC @trallard, do you have a preference for how to handle out-of-date links?